src/set_theory/game/impartial.lean

```diff
@@ -0,0 +1,221 @@
+/-
+Copyright (c) 2020 Fox Thomson. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Fox Thomson
+-/
+import set_theory.game.position
+import tactic.nth_rewrite.default
+import tactic
```

Comment:
Please reduce to just the tactic imports you need. (`tactic.basic` is fine)

Reply:
I don't seem to be able to import nth_rewrite though `tactic.basic`

Reply:
Ah, maybe we need to add `nth_rewrite` to `tactic.basic` (in a later PR).

Reply:
I just meant import `tactic.nth_rewrite` (or anything else outside `tactic.basic`) directly, rather than pulling in the kitchen sink with `tactic`.